src/input/mpegts/satip/satip.c

```diff
@@ -210,6 +224,13 @@ const idclass_t satip_device_class =
       .off      = offsetof(satip_device_t, sd_bindaddr),
     },
     {
+      .type     = PT_BOOL,
+      .id       = "disablefritz",
+      .name     = "Disable FRITZ!-specific workarounds",
```

Comment:
Great, but could you change this as more universal 'No firmware-specific workarounds' which will apply for all hacks ?